medical_prescription/tests/test_medical_prescription_order_line.py

```diff
@@ -113,6 +113,7 @@ def test_onchange_medical_medication_id(self):
             'medical_prescription.medical_prescription_order_order_line_2'
         )
         self.rx_line.medical_medication_id = rx_line2.medical_medication_id
+        self.rx_line._onchange_medical_medication_id()
```

Comment:
Woot! Always keep in mind that the first step in TDD is a broken build. Next step is a fixed build. If you are always 💚 , the code is not as perfect as the color alludes to